docker-compose.yml

```diff
@@ -9,3 +9,5 @@ services:
     image: nginx
     volumes:
       - ./nginx.conf:/etc/nginx/conf.d/default.conf:ro
+    ports:
+      - 80:80
```

Comment:
By convention, the docker-compose.yml should contain only the base configuration. It's recommended to define all the environment specific options in override files.
https://docs.docker.com/compose/extends/#multiple-compose-files